QuantLib/ql/termstructures/volatility/optionlet/optionletstripper.hpp

```diff
@@ -40,6 +40,7 @@ namespace QuantLib {
       public:
         //! \name StrippedOptionletBase interface
         //@{
+        enum Model{Normal, ShiftedLognormal};
```

Comment:
Yes, sure. I was just wondering if it's worth mentioning the "shifted" part or if it's kind of a given (since we're passing a displacement).